ibm/service/backuprecovery/resource_ibm_backup_recovery_connector_access_token.go

```diff
@@ -0,0 +1,162 @@
+// Copyright IBM Corp. 2025 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+/*
+ * IBM OpenAPI Terraform Generator Version: 3.94.0-fa797aec-20240814-142622
+ */
+
+package backuprecovery
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex"
+	"github.com/IBM/go-sdk-core/v5/core"
+	"github.com/IBM/ibm-backup-recovery-sdk-go/backuprecoveryv1"
+)
+
+func ResourceIbmBackupRecoveryConnectorAccessToken() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceIbmBackupRecoveryConnectorAccessTokenCreate,
+		ReadContext:   resourceIbmBackupRecoveryConnectorAccessTokenRead,
+		DeleteContext: resourceIbmBackupRecoveryConnectorAccessTokenDelete,
+		CustomizeDiff: checkDiffResourceIbmBackupRecoveryConnectorAccessToken,
+		Importer:      &schema.ResourceImporter{},
+
+		Schema: map[string]*schema.Schema{
+			"username": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "Specifies the login name of the Cohesity user.",
+			},
+			"password": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "Specifies the password of the Cohesity user account.",
+			},
+			"domain": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "Specifies the domain the user is logging in to. For a local user the domain is LOCAL. For LDAP/AD user, the domain will map to a LDAP connection string. A user is uniquely identified by a combination of username and domain. LOCAL is the default domain.",
+			},
+			"access_token": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Created access token.",
+			},
+			"privileges": &schema.Schema{
+				Type:        schema.TypeList,
+				Computed:    true,
+				Description: "Privileges for the user.",
+				Elem:        &schema.Schema{Type: schema.TypeString},
+			},
+			"token_type": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Access token type.",
+			},
+		},
+	}
+}
+
+func checkDiffResourceIbmBackupRecoveryConnectorAccessToken(context context.Context, d *schema.ResourceDiff, meta interface{}) error {
+	if d.Id() == "" {
+		return nil
+	}
+
+	for fieldName := range ResourceIbmBackupRecoveryConnectorAccessToken().Schema {
```

Comment:
Since we added Forcenew we don't need any this function "checkDiffResourceIbmBackupRecoveryConnectorAccessToken"